orca/request_opendap.py

```diff
@@ -1,21 +1,30 @@
 import re
-from xarray import open_dataset
+from xarray import open_dataset, load_dataset
+from tempfile import NamedTemporaryFile
 
+from .split import split_url
 
-def request_opendap(url, out_file):
-    try:
-        d = open_dataset(url)
-        d.to_netcdf(out_file)
-    except Exception as e:
-        raise e
 
-    return out_file
+def request_opendap(url):
+    data = open_dataset(url)
+    size = data.nbytes
+
+    if size / 2 > 5e8:
+        urls = split_url(url, size)
+    else:
+        urls = [url]
+
+    output = []
+    for path in urls:
+        tmp = NamedTemporaryFile(suffix=".nc", dir="/tmp")
+        open_dataset(path).to_netcdf(tmp.name)
+        output.append(tmp)
```

Comment:
I think some logging messages would be helpful here to follow the progress. You mentioned it can take some time so it would be good to have some feedback during the process to know it isn't just hanging.